test/stable/common/pvc.rb

```diff
@@ -3,8 +3,8 @@
 
```

Comment:
We should rename this file to pvc_spec.rb, otherwise it won't be picked up by the test framework. Should have caught that in the initial PR, sorry about that :(

Reply:
Ahh now I get where my confusion came from, thanks! :)

Reply:
fixed.